libs/server/Transaction/TransactionManager.cs

```diff
@@ -172,6 +175,13 @@ internal bool RunTransactionProc(byte id, ArgSlice input, CustomTransactionProce
                     return false;
                 }
 
+                if (state == TxnState.Aborted)
```

Comment:
I'm wondering if there should another state (CROSSLOTABORT?) when slot verification fails.

Reply:
Is it not difficult to add but how is this useful? When txn is aborted the client will receive a message that validation failed due slot verification issue.

Reply:
Currently, transaction fail possible only due to slot verification issue.  Can there be more reasons later? If yes, it would keep showing slot verification issue hence thought of another state.

Reply:
We return a state of aborted and the error message to the caller, so the caller can handle any type of abort.